### PR TITLE
Implemented Graphs for Hero Benchmarks

### PR DESCRIPTION
**Pull request by bdbch**
Not related to any issue, just thought this could be a good feature.

**Implemented Graphs for Hero Benchmarks:**
image

### CODE REVIEW

**Comment by bdbch** [comment]
Added some more spice! :)

image

**Comment by bdbch** [comment]
I also improved the histogram charts hover effect which was very... grayish?

image

File: src/components/Hero/BenchmarkGraphs.jsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    display: block;
+    margin-left: 0;
+    margin-right: 0;
+  }
+`;
+
+const mapData = (data, key) => data.map(item => ({ Percentage: (`${item.percentile * 100}%`), Value: Number(item[key].toFixed(2)) }));
+
+const getData = data => [
+  { title: 'Gold per Minute', data: mapData(data, 'gold_per_min'), color: constants.golden },
```
Comment (howardchung) [inline src/components/Hero/BenchmarkGraphs.jsx]: Should these strings be localized?

Comment (bdbch) [reply src/components/Hero/BenchmarkGraphs.jsx]: Right! I'll update that ASAP

**Comment by bdbch** [comment]
@howardchung localization is now added

File: src/components/Hero/BenchmarkGraph.jsx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+import {
+  XAxis,
+  YAxis,
+  Tooltip,
+  Area,
+  AreaChart,
+  CartesianGrid,
+  ResponsiveContainer,
+} from 'recharts';
+
```
Comment (howardchung) [inline src/components/Hero/BenchmarkGraph.jsx]: empty line?

File: src/components/Hero/BenchmarkGraphs.jsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    display: block;
+    margin-left: 0;
+    margin-right: 0;
+  }
+`;
+
+const mapData = (data, key) => data.map(item => ({ Percentage: (`${item.percentile * 100}%`), Value: Number(item[key].toFixed(2)) }));
+
+const getData = (data, strings) => [
+  { title: strings.benchmarks_gold_per_min, data: mapData(data, 'gold_per_min'), color: constants.golden },
```
Comment (howardchung) [inline src/components/Hero/BenchmarkGraphs.jsx]: I think you can reuse:
https://github.com/odota/web/blob/c39e25ec854f176ab8ee7bb130d98aad378d5e4c/src/lang/en-US.json#L922

**Comment by bdbch** [comment]
@howardchung resolved

File: src/lang/en-US.json [hunk 4]
```diff
@@ -118,6 +118,11 @@
   "barracks_value_2048": "Radiant Top Ranged",
 
   "benchmarks_description": "%d %s is equal or higher than %d% of recent performances on this hero",
+  "benchmarks_xp_per_min": "XP per Minute",
```
Comment (howardchung) [inline src/lang/en-US.json]: i think the rest of these strings also have existing versions that you could reuse as well.  Didn't notice it before I merged, if you want to you can submit a PR to do that @bdbch

**Pull request merged**